From 3646de78ed303e1c84c78b676859df9c2db33863 Mon Sep 17 00:00:00 2001
From: Lorenz Brun <lorenz@brun.one>
Date: Mon, 25 Jan 2021 18:20:01 +0100
Subject: [PATCH] Point CacheDir to the correct location for Metropolis

This is arguably an ugly hack, but they hardcoded it and the fastest way to
access anything resembling a config is through three different repos:
containernetworking/cni -> containerd/go-cni -> containerd/cri ->
containerd/containerd.
---
 libcni/api.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libcni/api.go b/libcni/api.go
index 5a43219..5d71426 100644
--- a/libcni/api.go
+++ b/libcni/api.go
@@ -38,7 +38,7 @@ import (
 )
 
 var (
-	CacheDir = "/var/lib/cni"
+	CacheDir = "/ephemeral/containerd/cni-cache"
 	// slightly awkward wording to preserve anyone matching on error strings
 	ErrorCheckNotSupp = fmt.Errorf("does not support the CHECK command")
 )
-- 
2.44.1

